target/linux/ramips/base-files/etc/diag.sh

```diff
@@ -123,8 +123,9 @@ get_status_led() {
 	pbr-d1)
 		status_led="$board:orange:power"
 		;;
+	f5d8235-v1|\
 	f5d8235-v2)
-		status_led="$board:blue:router"
```

Comment:
Man, you told me about it and I forgot it again :-(. Please add this detailed explanation to the commit message.